server/runtime/src/test/java/org/infinispan/server/configuration/JsonSerializationTest.java

```diff
@@ -125,6 +129,13 @@ public void testJsonSerialization() throws Exception {
       assertEquals("ldap://${org.infinispan.test.host.address}:10389", ldapRealm.get("url").asText());
       assertEquals("uid=admin,ou=People,dc=infinispan,dc=org", ldapRealm.get("principal").asText());
       assertEquals("***", ldapRealm.get("credential").asText());
+
+      /*JsonNode ldapNameRewriter = ldapRealm.get("name-rewriter");
+      JsonNode ldapRegexPrincipalTransformer = ldapNameRewriter.get("regex-principal-transformer");
+      assertEquals("uid", ldapRegexPrincipalTransformer.get("name").asText());
+      assertEquals("uid", ldapRegexPrincipalTransformer.get("pattern").asText());
+      assertEquals("uid", ldapRegexPrincipalTransformer.get("replacement").asText());*/
+
```

Comment:
Is this part not working?

Reply:
@tristantarrant Could you create a JIRA detailing the parts of the config that are not being serialized at the moment?

Reply:
https://issues.jboss.org/browse/ISPN-10961